dojo/settings/settings.dist.py

```diff
@@ -1345,6 +1345,7 @@ def saml2_attrib_map_format(din):
     "Red Hat Satellite": ["description", "severity"],
     "Qualys Hacker Guardian Scan": ["title", "severity", "description"],
     "Cyberwatch scan (Galeax)": ["title", "description", "severity"],
+    "Cycognito Scan": ["description"],
```

Comment:
```suggestion
    "Cycognito Scan": ["title", "severity"],
```
It looks like there are timestamps in the description field. This will likely mean that different reports will have a harder time matching findings. Would title/severity work?